src/components/SearchGridManualLoad.vue

```diff
@@ -94,7 +94,8 @@ export default {
     },
     _imagesCount() {
       const count = this.useInfiniteScroll ? this.$store.state.imagesCount : this.imagesCount;
-      return count.toLocaleString('en');
+      const localeCount = count.toLocaleString('en');
+      return localeCount === '10,000' ? `Over ${localeCount} images` : `${localeCount} images`;
```

Comment:
Doing number checking instead of string checking is better in this case. It's faster, and it prevents future problems in case the number changes for any reason. Can you change the check to `count >= 10000`?

Reply:
Fixed now!